### PR TITLE
Update powerVCF.psm1

### PR DESCRIPTION
**Pull request by kannanbalu**
Added Get-VCFCredentialTask 
(Addresses :
1.Get the Credentials Tasks
2. Get a Credentials Task
3. Get the Resource Credentials for Credentials Task)

Added Cancel-VCFCredentialTask
Added Retry-VCFCredentialTask (Addresses both retry of password rotate/update task)
Added Get-VCFUpgradables

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -1497,6 +1556,119 @@ Export-ModuleMember -Function Set-VCFCredential
...
+
+<#
+    .SYNOPSIS
+    Connects to the specified SDDC Manager and retry a failed rotate/update passwords task
+
+    .DESCRIPTION
+	The Retry-VCFCredentialTask cmdlet connects to the specified SDDC Manager and retry a failed rotate/update password task
+
+    .EXAMPLE
+	PS C:\> Retry-VCFCredentialTask -id -id 4d661acc-2be6-491d-9256-ba3c78020e5d -json .\Credential\updateCredentialSpec.json
```
Comment (GaryJBlake) [inline powerVCF.psm1]: There are 2 x -id in this example

File: powerVCF.psm1 [hunk 2]
```diff
@@ -1497,6 +1556,119 @@ Export-ModuleMember -Function Set-VCFCredential
...
+    This example shows how to update passwords of a resource type using a json spec
+
+    .EXAMPLE
+	PS C:\> Retry-VCFCredentialTask -id -id 4d661acc-2be6-491d-9256-ba3c78020e5d -json .\Credential\rotateCredentialSpec.json
```
Comment (GaryJBlake) [inline powerVCF.psm1]: Same here 2 x -id in the example

File: powerVCF.psm1 [hunk 3]
```diff
@@ -1497,6 +1556,119 @@ Export-ModuleMember -Function Set-VCFCredential
...
+#>
+
+	param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$privilegedUsername,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$privilegedPassword,
+        [Parameter (Mandatory=$true)]
```
Comment (GaryJBlake) [inline powerVCF.psm1]: Asking for $privilegedUsername and $privilegedPassword inputs as required but examples are not showing the inputs

File: powerVCF.psm1 (hunk 2 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 3 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Good catch. Fixed.

**Pull request merged**